docs/data-sources/phylop.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: PhyloP
+---
+
+import JSON from './phylop-json.md'
+
+## Overview
+
+PhyloP (phylogenetic p-values) conservation scores are obtained from the [PHAST package] (http://compgen.bscb.cornell.edu/phast/) for multiple alignments of vertebrate genomes to the human genome. For GRCh38, the multiple alignments are against 19 mammals and for GRCh37, it is against 45 vertebrate genomes.
+
+:::info Publication
+Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, Rosenbloom K, Clawson H, Spieth J, Hillier LW, Richards S, et al. Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. **Genome Res. 2005** Aug;15(8):1034-50.  (http://www.genome.org/cgi/doi/10.1101/gr.3715005)
+:::
+
+
+## File Parsing
+The data is provided in WigFix files which is a text file that provides conservation scores for contiguous intervals in the following format:
+```scss
+fixedStep chrom=chr1 start=10918 step=1
+0.064
+0.058
+0.064
+0.058
+0.064
+0.064
+fixedStep chrom=chr1 start=34045 step=1
+0.111
+0.100
+0.111
+0.111
+0.100
+0.111
+0.111
+0.111
+0.100
+0.111
+-1.636
+```
+We convert them to binary files with indexes for fast query. Note that these are scores for genomic positions and are reported only for SNVs.
+## Download URL
+
+[GRCh37](http://hgdownload.cse.ucsc.edu/goldenpath/hg19/phyloP46way/vertebrate/)
```

Comment:
Go ahead and output the entire URL rather than labeling them GRCh37 and GRCh38. Similar to dbSNP.